proto/lighthouse-result.proto

```diff
@@ -131,7 +132,8 @@ message LighthouseResult {
     // This enum served the emulated_form_factor field, but in v7, that field
     // was deprecated. Meanwhile in v7, the form_factor field was added and has
     // a mobile/desktop enum, so we simplify reuse, minus the `none` option.
-    // See https://github.com/GoogleChrome/lighthouse/blob/master/docs/emulation.md#changes-made-in-v7
+    // See
+    // https://github.com/GoogleChrome/lighthouse/blob/master/docs/emulation.md#changes-made-in-v7
```

Comment:
![image](https://user-images.githubusercontent.com/2301202/105125001-f5cc3480-5aa0-11eb-9071-ec82d50602c2.png)

Reply:
seriously though, how do all the internal google formatters that wind up touching lh/chrome-launcher have such bad wrapping rules? 😭 

clang-format and the markdown reformatter for devsite were similarly awful on the line wrap stuff.

Reply:
_sometimes_, the real opensource world is better than internal google tools. 😛